File: muridku/muridku.backend/GlobalDbProcessor/Muridku.QueryRequestReceiver/Models/Member.cs
using System;

namespace Muridku.QueryRequestReceiver.Models
{
  public class Member
  {
    public int id { get; set; }
    public string name { get; set; }
    public string address { get; set; }
    public DateTime? birth_dt { get; set; }
    public string birth_place { get; set; }
    public string mobile_phn { get; set; }
    public int? institution_id { get; set; }
    public int? faculty_id { get; set; }
    public string usr_crt { get; set; }
    public DateTime dtm_crt { get; set; }
    public string usr_upd { get; set; }
    public DateTime dtm_upd { get; set; }
  }
}

File: database.admin/database/app/Models/Member.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Member extends Model
{
    use HasFactory;

    protected $table = 'member';
    public $timestamps = false;

    protected $fillable = [
        'name',
        'address',
        'gender',
        'birth_dt',
        'birth_place',
        'mobile_phn',
        'status',
        'inst_type',
        'city_id',
        'institution_id',
        'faculty_id',
        'alpha_member_id',
        'usr_crt', 
        'dtm_crt', 
        'usr_upd', 
        'dtm_upd',
    ];

    public function getTypeName()
    {
        $typename = [
            'SHS' => 'SMA',
            'COL'  => 'Kampus',
            'WORK' => 'Kerja',
        ];

        return $typename[$this->inst_type] ?? 'Tipe Tidak Diketahui';
    }

    public function city()
    {
        return $this->belongsTo(City::class, 'city_id', 'id');
    }

    public function institution()
    {
        return $this->belongsTo(Institution::class, 'institution_id', 'id');
    }

    public function faculty()
    {
        return $this->belongsTo(Faculty::class, 'faculty_id', 'id');
    }
}
